frontend/javascripts/admin/job/job_list_view.js

```diff
@@ -140,15 +140,21 @@ class JobListView extends React.PureComponent<Props, State> {
   };
 
   renderState = (__: any, job: APIJob) => {
-    const stateString = _.capitalize(job.state.toLowerCase());
-    if (job.state === "SUCCESS") return stateString;
-    else {
-      return (
-        <Tooltip title="Something went wrong when executing this job. Feel free to contact us if you need assistance.">
-          {stateString}
-        </Tooltip>
-      );
-    }
+    const tooltipMessages = {
+      UNKNOWN:
+        "The status information for this job could not be retreived. Please try again in a few minutes, or contact us if you need assistance.",
+      SUCCESS: "This job has successfully been executed.",
+      PENDING: "This job will run as soon as a worker becomes available.",
+      STARTED: "This job is currently running.",
+      FAILURE:
+        "Something went wrong when executing this job. Feel free to contact us if you need assistance.",
+      MANUAL:
+        "The automatic job failed, but admins have been notified and will attempt to repair it manually. Please check back here soon.",
```

Comment:
Maybe:

```suggestion
        "The job will be handled by an admin shortly, since it couldn't be finished automatically. Please check back here soon.",
```

This puts emphasis on what's happening next and avoids the negative word "fail"